ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -372,11 +355,20 @@ AmbryReplica getReplicaForPartitionOnNode(String hostname, int port, String part
      */
     @Override
     public void onLiveInstanceChange(List<LiveInstance> liveInstances, NotificationContext changeContext) {
+      logger.trace("Live instance change triggered with: {}", liveInstances);
```

Comment:
perhaps leave a comment here saying that helix guarantees that ordering of callbacks is correct?